### PR TITLE
Update installation.md: Install using apt or rpm

### PR DESCRIPTION
**Pull request by autoantwort**
**What this PR does / why we need it**:
Currently you suggest to download the files manually instead of using apt/rpm which means you don't get any updates. I thing using apt/rpm would be better. 

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

I admit I have not read the full https://grafana.com/docs/writers-toolkit/ guide, seems a lot of work for such a small change 😅

### CODE REVIEW

**Comment by autoantwort** [comment]
Another problem is that after installing the binaries the examples does not work: https://community.grafana.com/t/promtail-open-var-log-boot-log-permission-denied/49202

File: docs/sources/send-data/promtail/installation.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ or there is a Helm chart to install it in a Kubernetes cluster.
 Every Grafana Loki release includes binaries for Promtail which can be found on the
 [Releases page](https://github.com/grafana/loki/releases) as part of the release assets. 
 
+## Install using APT or RPM package manager
+
+See [here](../../setup/install/local.md).
```
Comment (JStickler) [inline docs/sources/send-data/promtail/installation.md:22]:
```suggestion
See the instructions [here](https://grafana.com/docs/loki//setup/install/local/#install-using-apt-or-rpm-package-manager). 
```

Comment (autoantwort) [reply docs/sources/send-data/promtail/installation.md]: Is the `/latest/` inserted here automatically?

**Review by JStickler — APPROVED** [review]
[docs team] Thanks for the contribution, LGTM

**Pull request merged**